client-app/src/api/contact/AddressAPI.js

```diff
@@ -8,29 +8,15 @@ export default {
         const AUTH_TOKEN = 'Bearer ' + localStorage.getItem('access_token');
         axios.defaults.headers.common['Authorization'] = AUTH_TOKEN;
 
-        return axios
-            .post(requestUrl, address)
-            .then(function(response) {
-                return response.data.data;
-            })
-            .catch(function(error) {
-                return error.response;
-            });
+        return axios.post(requestUrl, address);
```

Comment:
Top opschoning, maar met AxiosInstance kun je ook de `AUTH_TOKEN` en `axios.defaults.headers...` op lijn 8 en 9 opruimen

Reply:
Cleaned up. In AddressAPI alle 4 de functies nu met AxiosInstance.